python/python/ert_gui/shell/simulations.py

```diff
@@ -33,9 +33,18 @@ def ensembleExperiment(self, line):
         simulation_runner = EnkfSimulationRunner(self.ert())
 
         now = time.time()
-
         print("Ensemble Experiment started at: %s" % datetime.now().isoformat(sep=" "))
 
+        print("Create the run path!")
+        iteration_count = self.ert().analysisConfig().getAnalysisIterConfig().getNumIterations()
```

Comment:
I'll do this as well, but I will keep the comment about the iteration_count so that the next person that looks as it will know that something is not yet watertight.